src/App.js

```diff
@@ -1,25 +1,34 @@
-import logo from './logo.svg';
 import './App.css';
+import { Header } from "./components/todo/Header";
+import { SectionMain } from "./components/todo/SectionMain";
+import React from "react";
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+export class App extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {todos: []};
+
+    }
+
+    handleTodos = (text) => {
+        const todo = {
+            done: false,
+            id: (Math.random() * 15.75).toFixed(2),
+        }
 
-export default App;
+        this.setState({
+            todos: [{...todo, text}, ...this.state.todos]
+        });
+
+        console.log({...todo, text})
```

Comment:
if this console log is not needed - remove it, try not to leave reduntant comments